docs/scripts_md/MakeNiiFilesBIDSCompliant.md

```diff
@@ -28,12 +28,13 @@ Available options are:
 # DESCRIPTION
 
 This \*\*BETA\*\* version script will create a BIDS compliant NIfTI file structure of
-the MINC files currently present in the \`assembly\` directory. If the argument
-\`tarchive\_id\` is specified, only the images from that archive will be processed.
-Otherwise, all files in \`assembly\` will be included in the BIDS structure,
-while looping though all the 'tarchive\_id\`'s in the 'tarchive\` table.
+the MINC files currently present in the `assembly` directory. If the argument
```

Comment:
Line 30, need to remove the backslashes.

Reply:
done